repositories.yaml

```
@@ -1,502 +1,460 @@
-    - mozphab/metrics.yaml
-  ping_files:
-    - mozphab/pings.yaml
-  dependencies:
-    - org.mozilla.deprecated:glean
-firefox-desktop:
-  app_id: firefox-desktop
-  description: The desktop version of Firefox
-  url: https://github.com/mozilla/gecko-dev
-  notification_emails:
-    - chutten@mozilla.com
-  metrics_files:
-    - toolkit/components/glean/metrics.yaml
-  ping_files:
-    - toolkit/components/glean/pings.yaml
-  dependencies:
-    - org.mozilla.deprecated:glean
-firefox-for-echo-show:
-  app_id: org-mozilla-connect-firefox
-  description: Firefox for Amazon's Echo Show
-  url: https://github.com/mozilla-mobile/firefox-echo-show
-  notification_emails:
-    - tlong@mozilla.com
-  metrics_files:
-    - app/metrics.yaml
-  dependencies:
-    - org.mozilla.components:service-glean
-firefox-reality-pc:
-  app_id: org-mozilla-firefoxreality
-  description: Firefox Reality for PC-connected VR platforms
-  url: https://github.com/MozillaReality/FirefoxRealityPC
-  notification_emails:
-    - dmu@mozilla.com
-  metrics_files:
-    - Source/FirefoxRealityUnity/metrics.yaml
-  ping_files:
-    - Source/FirefoxRealityUnity/pings.yaml
-  dependencies:
-    - glean-core
-mach:
-  app_id: mozilla-mach
-  description: Mach build telemetry
-  url: https://github.com/mozilla/gecko-dev
-  notification_emails:
-    - mhentges@mozilla.com
-  metrics_files:
-    - python/mach/metrics.yaml
-  ping_files:
-    - python/mach/pings.yaml
-  dependencies:
-    - org.mozilla.deprecated:glean
-glean-js:
-  app_id: glean-js-tmp
-  description: Experimentation with Glean.js
-  url: https://github.com/brizental/gleanjs
-  prototype: true
-  notification_emails:
-    - brizental@mozilla.com
-    - aplacitelli@mozilla.com
-    - mdroettboom@mozilla.com
-  branch: main
-  metrics_files:
-    - metrics.yaml
-  dependencies:
-    - glean-core
-  retention_days: 90
-firefox-focus-ios:
-  app_id: org-mozilla-ios-focus
-  description: Firefox Focus on iOS. Klar is the sibling application
-  url: https://github.com/mozilla-mobile/focus-ios
-  notification_emails:
-    - jboek@mozilla.com
-    - tlong@mozilla.com
-  branch: main
-  metrics_files:
-    - Blockzilla/metrics.yaml
-  dependencies:
-    - glean-core
-  retention_days: 720
-firefox-klar-ios:
-  app_id: org-mozilla-ios-klar
-  description: Firefox Klar on iOS. Focus is the sibling application
-  url: https://github.com/mozilla-mobile/focus-ios
-  notification_emails:
-    - jboek@mozilla.com
-    - tlong@mozilla.com
-  branch: main
-  metrics_files:
-    - Blockzilla/metrics.yaml
-  dependencies:
-    - glean-core
-  retention_days: 720
+libraries:
+  - v1_name: glean-core
+    description: Modern cross-platform telemetry (core library)
+    notification_emails:
+      - frank@mozilla.com
+      - mdroettboom@mozilla.com
+    url: https://github.com/mozilla/glean
+    branch: main
+    metrics_files:
+      - glean-core/metrics.yaml
+    ping_files:
+      - glean-core/pings.yaml
+    library_names:
```

Looking at this file, I wonder if `library_names` is redundant?

I might also suggest putting this field on top.

```suggestion
    names:
```

---

I want the field name to be fairly self-documenting; I worry that `names` is too general and ambiguous. The real purpose of this field is to provide a link between the names given by apps in their `dependencies` field, so it might be nice to give this some name related to "dependency". Something like `dependency_names`, `declares_dependencies`, `defines_dependencies`, etc.

I think I like `defines_dependencies`. It's a bit verbose, but seems much more descriptive about the purpose of the field.